experiments/dynamic-benches/src/network_latency.rs

```
@@ -1266,14 +1231,12 @@ pub mod ppindexed {
         }
 
         fn receive_network(&mut self, msg: NetMessage) -> () {
-            trace!(self.ctx.log(), "Ponger received msg {:?}", msg,);
-            let sender = msg.sender;
-            match msg.data.try_deserialise::<Ping, Ping>() {
+            trace!(self.ctx.log(), "Ponger received msg {:?}", msg, );
+            let sender = msg.sender.clone();
+            match msg.try_deserialise::<Ping, Ping>() {
                 Ok(ping) => {
                     trace!(self.ctx.log(), "Ponger got Ping!");
-                    sender
-                        .tell_serialised(Pong::new(ping.index), self)
-                        .expect("serialise");
+                    sender.tell_serialised(Pong::new(ping.index), self).ok();
```

Why are we discarding errors here?

---

My bad, changed to expect again